### PR TITLE
statistic: add the statistic of flow

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve? 
Add the metrics of the flow statistics of the store. 

### What is changed and how it works?
Add metrics.

### Check List 

Tests 

 - Manual test

### CODE REVIEW

File: server/store_statistics.go [hunk 1]
```diff
@@ -101,6 +101,14 @@ func (s *storeStatistics) Observe(store *core.StoreInfo) {
 	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_available").Set(float64(store.GetAvailable()))
 	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_used").Set(float64(store.GetUsedSize()))
 	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_capacity").Set(float64(store.GetCapacity()))
+
+	// Store flows.
+	storeBytesWriteRate, storeBytesReadRate := store.GetRollingStoreStats().GetBytesRate()
+	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_bytes_write_rate").Set(float64(storeBytesWriteRate))
```
Comment (shafreeck) [inline server/store_statistics.go]: According to the best practice of Prometheus's name ruling, is `store_write_rate_bytes`  preferred than `store_bytes_write_rate`?

Refer to: https://prometheus.io/docs/practices/naming/#metric-names

File: server/store_statistics.go [hunk 2]
```diff
@@ -101,6 +101,14 @@ func (s *storeStatistics) Observe(store *core.StoreInfo) {
...
+	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_bytes_read_rate").Set(float64(storeBytesReadRate))
+	storeKeysWriteRate, storeKeysReadRate := store.GetRollingStoreStats().GetKeysWriteRate(), store.GetRollingStoreStats().GetKeysReadRate()
+	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_keys_write_rate").Set(float64(storeKeysWriteRate))
```
Comment (shafreeck) [inline server/store_statistics.go]: store_write_rate_keys, using keys as the unit

File: server/store_statistics.go [hunk 3]
```diff
@@ -101,6 +101,14 @@ func (s *storeStatistics) Observe(store *core.StoreInfo) {
 	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_available").Set(float64(store.GetAvailable()))
 	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_used").Set(float64(store.GetUsedSize()))
 	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_capacity").Set(float64(store.GetCapacity()))
+
+	// Store flows.
+	storeBytesWriteRate, storeBytesReadRate := store.GetRollingStoreStats().GetBytesRate()
```
Comment (rleungx) [inline server/store_statistics.go]: prefer to use a variable for `GetRollingStoreStats`

File: server/store_statistics.go (hunk 1 above)
Comment (nolouch) [reply server/store_statistics.go]: I think the name is the display name, not variable name.

**Pull request merged**